dev/io.openliberty.jakarta.jsonp.2.1_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -105,56 +90,6 @@
 	<build>
 		<directory>${targetDirectory}</directory>
 		<plugins>
-			<!-- Plugin used to copy external artifacts to signaturedirectory for signature tests -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-dependency-plugin</artifactId>
-				<version>3.2.0</version>
-				<executions>
-					<execution>
-						<id>copy</id>
-						<phase>generate-test-sources</phase>
-						<goals>
-							<goal>copy</goal>
-						</goals>
-						<configuration>
-							<artifactItems>
-								<!-- This artifact is being copied from the system instead	-->							
-								<artifactItem>
-									<groupId>jakarta.json</groupId>
-									<artifactId>jakarta.json-api</artifactId>
-									<version>${jakarta.json.version}</version>
-									<type>jar</type>
-									<overWrite>true</overWrite>
-									<outputDirectory>${project.build.directory}/signaturedirectory</outputDirectory>
-									<destFileName>jakarta.json-api.jar</destFileName>
-								</artifactItem>
-							</artifactItems>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-			<!-- Plugin used to internal bundles to signaturedirectory for signature tests -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-			    <artifactId>maven-antrun-plugin</artifactId>
-			    <version>3.0.0</version>
-			    <executions>
-			        <execution>
-			            <id>copy-jsonp</id>
-			            <phase>generate-test-sources</phase>
-			            <goals>
-			                <goal>run</goal>
-			            </goals>
-			            <configuration>
-			              <target>
-<!-- 			                <copy file="${io.openliberty.jakarta.jsonp.3.0}" tofile="${project.build.directory}/signaturedirectory/jakarta.json.bind-api.jar"/> -->
-			                <copy file="${io.openliberty.jakarta.jsonp.2.1}" tofile="${project.build.directory}/signaturedirectory/jakarta.json-api.jar"/>
-			              </target>
-			            </configuration>
-			        </execution>
-			    </executions>
-			</plugin>
```

Comment:
I think you may have accidentally deleted both of these plugins, when only one needed to be removed. 
We should be using the plugin with: 
<!-- Plugin used to internal bundles to signaturedirectory for signature tests -->